File: src/main/java/com/quanweng/shopping/pojo/UserTraceReqInfo.java
package com.quanweng.shopping.pojo;

import lombok.Data;

@Data
public class UserTraceReqInfo {
    private String requestSessionID; // 请求唯一标识
    private String userId;           // 用户ID或NO_LOGIN
    private String reqHeader;        // 完整请求头(JSON字符串)
}

File: src/main/java/com/quanweng/shopping/pojo/UserTraceReqInfo.kt
package com.quanweng.shopping.pojo

data class UserTraceReqInfo(
    var requestSessionID: String = "", // 請求唯一標識
    var userId: String = "NO_LOGIN",   // 使用者ID或NO_LOGIN
    var reqHeader: String = ""         // 完整請求頭(JSON or 純文字)
)
